### PR TITLE
Style Conference and Group Cards

### LINKED ISSUE #624: Style Conference Page to have the conference Card above the groups.

### PR DESCRIPTION
**Pull request by maxwolffe**
Fix #624

### CODE REVIEW

**Comment by warrenshen** [comment]
Merge conflicts!

File: app/assets/javascripts/components/group/group_page.es6.jsx [hunk 1]
```diff
@@ -56,8 +56,9 @@ class GroupPage extends Component {
   render() {
     return (
       <div style={this.styles.container}>
-      <Header hasSidebar={true}/>
+      <Header hasSidebar={true} active={true}/>
```
Comment (warrenshen) [inline app/assets/javascripts/components/group/group_page.es6.jsx]: This prop is outdated now.

File: app/assets/javascripts/components/group/group_page.es6.jsx [hunk 2]
```diff
@@ -56,8 +56,9 @@ class GroupPage extends Component {
...
         <Sidebar
+          active={true}
           hidden={this.state.sidebar}
```
Comment (warrenshen) [inline app/assets/javascripts/components/group/group_page.es6.jsx]: Hidden is also outdated now.

File: app/assets/javascripts/components/group/group_page.es6.jsx [hunk 3]
```diff
@@ -56,8 +56,9 @@ class GroupPage extends Component {
   render() {
     return (
       <div style={this.styles.container}>
-      <Header hasSidebar={true}/>
+      <Header hasSidebar={true} active={true}/>
         <Sidebar
```
Comment (warrenshen) [inline app/assets/javascripts/components/group/group_page.es6.jsx]: Can you also remove the `TODO` in `sidebar.es6.jsx`.

File: app/assets/javascripts/stores/group_store.es6.jsx [hunk 4]
```diff
@@ -4,6 +4,7 @@
     constructor() {
       this.group = {
         students: [],
+        conference: {},
```
Comment (warrenshen) [inline app/assets/javascripts/stores/group_store.es6.jsx]: Alphabetize!

File: app/serializers/group_index_serializer.rb [hunk 5]
```diff
@@ -1,5 +1,6 @@
 class GroupIndexSerializer < GroupBaseSerializer
 
   has_many :students, serializer: StudentBaseSerializer
+  has_one :conference, serializer: ConferenceBaseSerializer
```
Comment (warrenshen) [inline app/serializers/group_index_serializer.rb]: Add space above this line!

File: app/assets/javascripts/stores/conference_store.es6.jsx [hunk 6]
```diff
@@ -9,9 +9,8 @@
       };
       this.sidebar = true;
       this.conference = {
-        conference: {
-          groups: [],
-        },
+        id: 1,
```
Comment (warrenshen) [inline app/assets/javascripts/stores/conference_store.es6.jsx]: Need this?

File: app/serializers/group_show_serializer.rb [hunk 7]
```diff
@@ -1,5 +1,6 @@
 class GroupShowSerializer < GroupIndexSerializer
 
-  has_many :students, serializer: StudentGroupSerializer
+	has_many :students, serializer: StudentGroupSerializer
```
Comment (warrenshen) [inline app/serializers/group_show_serializer.rb]: Spacing.

File: app/serializers/group_show_serializer.rb [hunk 8]
```diff
@@ -1,5 +1,6 @@
...
+  has_one :conference, serializer: ConferenceBaseSerializer
```
Comment (warrenshen) [inline app/serializers/group_show_serializer.rb]: Add space above this line.

File: db/seeds/development/6_groups.rb [hunk 9]
```diff
@@ -1,4 +1,4 @@
-@alphabet = ("a".."z").to_a
+@alphabet = ("A".."Z").to_a
```
Comment (warrenshen) [inline db/seeds/development/6_groups.rb]: Single quotes for strings that don't have interpolation!

File: app/assets/javascripts/stores/conference_store.es6.jsx (hunk 6 above)
Comment (maxwolffe) [inline app/assets/javascripts/stores/conference_store.es6.jsx]: nope!

**Pull request merged**